source/frontend/src/features/properties/map/activity/detail/ActivityPropertyModal.tsx

```diff
@@ -58,16 +61,19 @@ export const ActivityPropertyModal: React.FunctionComponent<
   };
 
   const handleOk = async () => {
-    if (activityModel && allProperties?.length > 0) {
-      activityModel.actInstPropFiles =
-        selectedFileProperties.map(p => ({
-          activityId: activityModel?.id,
-          propertyFileId: p?.id,
-        })) ?? [];
-      const updatedActivity = await onSave(activityModel);
-      // if the activity was updated successfully, hide the modal.
-      if (!!updatedActivity) {
-        setDisplay(false);
+    if (disableSave) setDisplay(false);
+    else {
+      if (activityModel && allProperties?.length > 0) {
+        activityModel.actInstPropFiles =
+          selectedFileProperties.map(p => ({
+            activityId: activityModel?.id,
+            propertyFileId: p?.id,
+          })) ?? [];
+        const updatedActivity = await onSave(activityModel);
+        // if the activity was updated successfully, hide the modal.
+        if (!!updatedActivity) {
+          setDisplay(false);
+        }
```

Comment:
Looking at the whole logic this "stops" the bug by closing the dialog without saving but this might be confusing to the user who was able to interact with the property list via checkboxes. This is kinda of a gray area but I would lean towards implementing what Bobbi suggested in JIRA:

> I don't think we want that as a fix, because the users will still need to click that to view what properties are included in the activity.
> We could turn it into a view-only kind of button though, by not allowing them to select/deselect. (ie: no checkboxes)

What are your thoughts @devinleighsmith ?

Reply:
agreed, this doesn't fix the issue in my mind as the user will still see those property rows as selected even if they leave and go back to the modal. Unless the user is savvy and notices that no success message is displayed they will have no idea that their changes were not saved.

I would say either the checkboxes should be disabled but still visible as suggested.

Reply:
Added disables boolean to table to allow disable ability of checkbox.